FreeRTOS_DNS.c

```diff
@@ -431,7 +266,7 @@
         }
 
 /** @brief The list of all callback structures. */
-        static List_t xCallbackList;
+        _static List_t xCallbackList;
```

Comment:
Why this change?

Reply:
_static is #defined as static. But for unit-testing and cbmc checks, _static is defined away to nothing. This helps in covering all branches of the source code and generating scenarios that are not interdependent.
Such as, some test cases need the `xCallbackList` to be `NULL`. But once a function that initializes the list is called, we cannot test the NULL conditions later on...